### DISCUSSION TITLE
Can I save the log under the Log tab as a continuously growing external file?

### DISCUSSION THREAD

**Discussion by maxigaz** [falkTX/Carla#1622] [Q&A] [answered]
I've found a bug that happens when Zyn-Fusion is used together with Carla and I'm looking for a backtrace. I've tried looking at the terminal output of `carla` and `carla --gdb`, but I don't think its contents would be enough for a proper bug report.

There's also the Log tab in Carla, which has the option to save the log into a separate file, but when the bug is triggered, Carla crashes immediately, so I can't use that.

Is there a way to redirect what's on the log tab to an external file continuously? I don't know if I can actually find a backtrace there, but I thought it might be worth a try.

**Comment by falkTX** [comment, marked as answer]
Just disable the console logs in the settings, and all output will be sent to terminal as usual
